src/threads-api.ts

```diff
@@ -55,16 +41,6 @@ export class ThreadsAPI {
         accept:
           'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7',
         'accept-language': 'ko,en;q=0.9,ko-KR;q=0.8,ja;q=0.7',
-        pragma: 'no-cache',
-        referer: 'https://www.instagram.com/',
-        'sec-fetch-dest': 'document',
-        'sec-fetch-mode': `navigate`,
-        'sec-fetch-site': `cross-site`,
-        'sec-fetch-user': `?1`,
-        'upgrade-insecure-requests': `1`,
-        'x-asbd-id': undefined,
-        'x-fb-lsd': undefined,
-        'x-ig-app-id': undefined,
```

Comment:
Just a quick question - why are we removing HTTP headers?
This was for stability in case Threads adds some bot checks like LinkedIn.